[Service Offloading] Minimize after start screen capture

Chikdolman opened the pull request:
Minimize activity after allow screen capture
If disallow, the activity is terminated only for Service Offloading

Sunwoo-Nam commented on media/capture/content/android/java/src/org/chromium/media/ScreenOffloading.java:
```diff
@@ -193,11 +194,16 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
             mResultData = data;
             changeCaptureStateAndNotify(CaptureState.ALLOWED);
+            if (OffloadingUtils.IsServiceOffloading()) {
+                ApplicationStatus.getLastTrackedFocusedActivity().moveTaskToBack(true);
```
Just my opinion. How about notify that screen capture is started using Toast?

Chikdolman replied:
@Sunwoo-Nam  We have screen recording msg on top of android once recording starts. Therefore, I think we don't need toast.

Sunwoo-Nam reviewed (approved):
LGTM

Chikdolman committed:
[Service Offloading] Minimize after start screen capture

Minimize activity after allow screen capture
If disallow, the activity is terminated only for Service Offloading
